### ISSUE TITLE
Scss with @import statements does not save if dir path has [] chars

### ISSUE THREAD

**Issue by jayhung** [bdkjones/CodeKit-1-Old#652]
If dir path is /sites/[myproject]/css then a scss file with @import statements does not save.

![image](https://f.cloud.github.com/assets/531273/907245/34e64554-fd1e-11e2-8338-3d071391501c.png)

If the dir path stays the same, but the @import statements are removed, any remaining styles do compile and save correctly.

If the @import statements are kept in the file, and the path is changed to remove the [] chars, the styles do compile and save correctly.

**Comment by bdkjones** [comment]
You're using brackets in a project's name?

**Comment by jayhung** [comment]
Yes, in certain cases non alphanumeric chars are used (such as _, !, etc)
to control finder's sort order. Both underscores and exclamation marks work
well with codekit already.
